core/src/main/java/me/deadlight/ezchestshop/EzChestShop.java

```diff
@@ -49,7 +49,7 @@ public void onLoad() {
     public void onEnable() {
 
         plugin = this;
-        logConsole("&c[&eEzChestShop&c] &aEnabling EzChestShop - version " + this.getDescription().getVersion());
+        logConsole("&c[&eEzChestShop&c] &aEnabling EzChestShop - version " + this.getDescription().getVersion() + " Balda_");
```

Comment:
Remove your name for the PR, it's nice for testing, but should be removed for production.